api/analysis.go

```diff
@@ -119,7 +119,7 @@ func (h AnalysisHandler) AppLatest(ctx *gin.Context) {
 // @description Resources do not include relations.
 // @tags analyses
 // @produce json
-// @success 200 {object} []api.Analyses
+// @success 200 {object} []api.Analysis
```

Comment:
Understood that this method is an action on multiple Analyses, but we don't have defined that as special type, so swagger failed to re-generate its specs. Modified it to an "array of analysis" ```[]api.Analysis``` (same in other changes in analysis.go file).